### DISCUSSION TITLE
Argument 1 passed to Symfony\Component\HttpFoundation\InputBag::get() must be of the type string, null given

### DISCUSSION THREAD

**Discussion by SyedGhazanfar** [symfony/symfony#47550] [Symfony Book: The Fast Track] [answered]
``` Argument 1 passed to Symfony\Component\HttpFoundation\InputBag::get() must be of the type string, null given, called in D:\Syed Ghazanfar\Futafut\Backend\v2\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php on line 157 ```

Hi, I got this error in my laravel project, Basically I moving laravel 5.8 to laravel 7.29 then I received this error,

can anyone help me in this error.

**Comment by stof** [comment, marked as answer]
The bug happens in Laravel, not in Symfony. Please report it to them.

Passing `null` as parameter name in `InputBag::get()` has never been a supported use case (it was documented as `string` since Symfony 2.0.0 on the bag class used at that time). So if their middleware passes `null`, it has a bug.

**Reply by SyedGhazanfar** [reply]
Thanks Dear
